components/brave_rewards/resources/tip/components/siteBanner.tsx

```diff
@@ -138,42 +137,49 @@ class Banner extends React.Component<Props, State> {
   }
 
   getTweetText () {
-    if (!this.props.tweetMetaData ||
-        !this.props.tweetMetaData.tweetText ||
-        this.props.tweetMetaData.tweetText.length === 0) {
+    if (!this.props.mediaMetaData || this.props.mediaMetaData.mediaType !== 'twitter') {
```

Comment:
could we save `this.props.mediaMetaData` into `mediaMetaData` so that it's cleaner

Reply:
done